proto/pdpb.proto

```diff
@@ -627,4 +627,5 @@ message SyncMaxTSRequest {
 message SyncMaxTSResponse {
     ResponseHeader header = 1;
     Timestamp max_local_ts = 2;
+    repeated string synced_dcs = 3;
```

Comment:
Maybe just `dcs`, for the first phase they do not sync anything.

Reply:
nice idea.